### PR TITLE
Added German translation

### PR DESCRIPTION
**Pull request by aaronkerckhoff**
Since I'm a native German speaker, I decided to add a German translation to the locales.

### CODE REVIEW

**Review by sy-b — APPROVED** [review]

**Review by FlorianPanten — CHANGES_REQUESTED** [review]
Just some improvement suggestions

File: Extensions/combined/_locales/de/messages.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      "message": "Spenden"
+  },
+  "linkHelp": {
+      "message": "Hilfe"
+  },
+  "legendSettings": {
+      "message": "Einstellungen"
+  },
+  "textSettings": {
+      "message": "Likes/Dislikes nicht einreichen"
```
[the remark this answers is not in the document]
Comment (aaronkerckhoff) [reply Extensions/combined/_locales/de/messages.json:27]: Yes, sounds better. Thank you

File: Extensions/combined/_locales/de/messages.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "extensionName": {
+      "message": "Return YouTube Dislike"
+  },
+  "extensionDesc": {
+      "message": "Bringt die Möglichkeit zurück, Dislikes zu sehen"
```
[the remark this answers is not in the document]
Comment (aaronkerckhoff) [reply Extensions/combined/_locales/de/messages.json:6]: This actually was my first version but I then decided to change it literally. But yes, maybe this is better.

File: Extensions/combined/_locales/de/messages.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  },
+  "textDeveloper": {
+      "message": "von Dmitry Selivanov & Community"
+  },
+  "linkWebsite": {
+      "message": "Website"
```
[the remark this answers is not in the document]
Comment (aaronkerckhoff) [reply Extensions/combined/_locales/de/messages.json:12]: I thought about that, but I thought that this sounds a bit too formal. Personally, I don't know anyone who says "Webseite". But yes, maybe that still would be better.

**Comment by aaronkerckhoff** [comment]
> Just some improvement suggestions

Thank you very much for the suggestions. I changed the first two and currently considering the third one. Maybe another German native speaker can say how they'd rather translate "Website".

File: Extensions/combined/_locales/de/messages.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (chludek) [reply Extensions/combined/_locales/de/messages.json:12]: I would definitely still use "Website". As you said, "Webseite" sounds way too formal. If you really take it this far, you probably would have to translate words like "FAQ" or "Likes/Dislikes" as well. The remaining translations look also good for me 👍

**Review by chludek — APPROVED** [review]

File: Extensions/combined/_locales/de/messages.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (aaronkerckhoff) [reply Extensions/combined/_locales/de/messages.json:12]: Thank you very much for another opinion, so I think I'm going to leave it as "Website".

**Review by FlorianPanten — APPROVED** [review]

**Review by sy-b — APPROVED** [review]

**Pull request merged**